lib/src/signed_video_tlv.c

```diff
@@ -236,7 +236,7 @@ encode_general(signed_video_t *self, uint8_t *data)
   if (gop_info->has_timestamp) {
     data_size += sizeof(timestamp);
   }
-
+  data_size += self->sign_data->hash_size;
```

Comment:
Since you add new data to the serialization, please update the "Value fields" in the comments above.